src/main/java/com/redhat/devtools/intellij/commonuitest/fixtures/dialogs/FlatWelcomeFrame.java

```diff
@@ -111,12 +129,109 @@ public void clearWorkspace() {
     public void clearExceptions() {
         try {
             ideErrorsIcon().click();
+            find(IdeFatalErrorsDialog.class, Duration.ofSeconds(10)).clearAll();
         } catch (WaitForConditionTimeoutException e) {
-            LOGGER.log(Level.SEVERE, e.getMessage(), e);
-            return;
+            LOGGER.log(Level.INFO, e.getMessage(), e);
+
+            try {
+                find(IdeFatalErrorsDialog.class, Duration.ofSeconds(10)).clearAll();
+            } catch (Exception e2) {
+                LOGGER.log(Level.INFO, e.getMessage(), e2);
+            }
+        }
+    }
+
+    /**
+     * Open the 'Preferences' dialog
+     */
+    public void openSettingsDialog() {
+        if (intelliJVersion.toInt() <= 20202) {
```

Comment:
ideaVersion=intellijVersion.toInt()

Reply:
Well, I have overlooked that, fixed in whole class.